### PR TITLE
Removes duplicate 'the's

### LINKED ISSUE #3234: Duplicate "the" (coming from `the_ofThe`, etc.?)
From my investigation in #3199, I noticed some `UnitalChunk`s with sentences built using `the_ofThe`, etc. but also having "the" prepended (these specific examples from DblPendulum):

https://github.com/JacquesCarette/Drasil/blob/1831438948f4bb92ba7050a27ccbff6489904784/code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/Unitals.hs#L90-L92
https://github.com/JacquesCarette/Drasil/blob/1831438948f4bb92ba7050a27ccbff6489904784/code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/Unitals.hs#L98-L100
https://github.com/JacquesCarette/Drasil/blob/1831438948f4bb92ba7050a27ccbff6489904784/code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/Unitals.hs#L58-L60
https://github.com/JacquesCarette/Drasil/blob/1831438948f4bb92ba7050a27ccbff6489904784/code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/Unitals.hs#L146-L148

I believe this results in "the" being duplicated:
image

### PR DESCRIPTION
**Pull request by ManvendraJani**
closes #3234 

Note: 
The 'term' and definition of `UnitalChunk` are being abused somewhere in the code, for example currently the 'term' for lenRod_1 is 
```Haskell
len `ofThe` firstRod
```
and the definition  is defined as
```Haskell
len `the_ofThe` firstRod
```
but use cases for both appear the same in the generated files. I believe this is because the definition is never actually being used. My question was if an issue should be raised about this or is this something we aim to clean up along with the [Chunk Investigation](https://github.com/JacquesCarette/Drasil/projects/10)

### CODE REVIEW

**Review by smiths — APPROVED** [review]
The changes to stable look good to me.  As usual, I haven't reviewed how the changes were made, but the generated files look good.  :smile:

**Review by samm82 — COMMENTED** [review]
Skimming the changes to the Drasil code, these changes look good. However, the changes from #3455 seem to also be present here. The "proper" solution would be to revert those commits from this branch so that only the commits relevant to the "the" issue are present. An alternative would be to mark this as a draft until #3455 is merged in.

Each PR should be as small as possible to make it easy to see exactly what was changed. In future, make sure to be branching off of `master` instead of another branch (unless you specifically need changes from a branch to build off of them)!

File: code/drasil-data/lib/Data/Drasil/Concepts/Documentation.hs [hunk 1]
```diff
@@ -173,6 +173,7 @@ module_         = nc "module"         (cn'    "module"             )
 model           = nc "model"          (cn'    "model"              )
 name_           = nc "name"           (cn'    "name"               )
 nonfunctional   = nc "non-functional" (cn'    "non-functional"     ) --FIXME: Adjective
+note_            = nc "note"           (cn'    "note"               )
```
Comment (JacquesCarette) [inline code/drasil-data/lib/Data/Drasil/Concepts/Documentation.hs:176]: nitpick: line it up properly!

File: code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs [hunk 2]
```diff
@@ -83,7 +86,7 @@ makeDerivCons (E e) = EqnBlock e
 makeDerivCons s     = Paragraph s
 
 -- | Synonym for easy reading. Model rows are just 'String',['Contents'] pairs.
-type ModRow = [(String, [Contents])]
+type ModRow = [(Sentence, [Contents])]
```
Comment (JacquesCarette) [inline code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs:89]: I don't know if we want the first bit of a `ModRow` to be a full `Sentence`. Why is this change needed?

File: code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs [hunk 3]
```diff
@@ -213,17 +214,16 @@ descPairs IncludeUnits =
...
-  show RefBy             = "RefBy"
-  show Source            = "Source"
-  show Input             = "Input"
-  show Output            = "Output"
-  show InConstraints     = "Input Constraints"
-  show OutConstraints    = "Output Constraints"
-  show DefiningEquation  = "Equation"
-  show (Description _ _) = "Description"
-  show Notes             = "Notes"
+getSent :: Field -> Sentence
```
Comment (JacquesCarette) [inline code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs:217]: I think the source of the problem is here: `getSent` goes immediately into "too big" a data-structure.

File: code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs (hunk 2 above)
Comment (JacquesCarette) [reply code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs:89]: Also, these changes shouldn't be in this PR at all, they are about something other than duplicate 'the'.

File: code/stable/dblpendulum/SRS/HTML/DblPendulum_SRS.html [hunk 4]
```diff
@@ -167,22 +167,22 @@ <h2>Table of Symbols</h2>
                 </tr>
                 <tr>
                   <td><em>a<sub>x1</sub></em></td>
-                  <td>The horizontal acceleration of the first object</td>
+                  <td>Horizontal acceleration of the first object</td>
```
Comment (JacquesCarette) [inline code/stable/dblpendulum/SRS/HTML/DblPendulum_SRS.html:170]: So this change is by design? I like it, I prefer this without the extra leading `The`.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]

File: code/stable/dblpendulum/SRS/HTML/DblPendulum_SRS.html (hunk 4 above)
Comment (ManvendraJani) [reply code/stable/dblpendulum/SRS/HTML/DblPendulum_SRS.html:170]: This was the only change that was supposed to be committed, I accidentally branched off of my working branch for another issue.

**Comment by ManvendraJani** [comment]
@samm82 is it ok to just merge this into the changes from #3455?
